src/utils/automation-props.js

```diff
@@ -44,7 +44,8 @@ const getAutomationProps = (multiUserAutomation, getAutomations) => ({
     translation: "Object.ActionButton.Automation.Link",
     component: "link",
     url: (data) => `/automations/editor/${data.automationId}`,
-    show: (data) => data.automationId.length > 1 && multiUserAutomation,
+    show: (data) =>
+      Object.prototype.hasOwnProperty.call(data, "automationId") && data.automationId.length > 1 && multiUserAutomation,
```

Comment:
Can't you just check existence of automationId with ?

Reply:
I will update this